### PR TITLE
[Feature] Introduce new is_editable property

### PR DESCRIPTION
**Pull request by shpran**
# [ADF-1828](https://oat-sa.atlassian.net/browse/ADF-1828)

## Changes
* Added new property in RDF schema to be able to mark property as editable/non-editable

## Companion PRs
* https://github.com/oat-sa/tao-core/pull/4135
* https://github.com/oat-sa/extension-tao-test/pull/486

[ADF-1828]: https://oat-sa.atlassian.net/browse/ADF-1828?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: migrations/Version202411200904082348_generis.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+final class Version202411200904082348_generis extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Sync models';
+    }
+
+    public function up(Schema $schema): void
+    {
+        OntologyUpdater::syncModels();
```
Comment (gabrielfs7) [inline migrations/Version202411200904082348_generis.php:23]: - [ ] Lets remove this migrations before merge, please. We will not need this anymore

File: core/GenerisRdf.php [hunk 2]
```diff
@@ -31,6 +31,7 @@ interface GenerisRdf
     public const PROPERTY_ALIAS = 'http://www.tao.lu/Ontologies/generis.rdf#alias';
     public const PROPERTY_IS_LG_DEPENDENT = 'http://www.tao.lu/Ontologies/generis.rdf#is_language_dependent';
     public const PROPERTY_IS_STATISTICAL = 'http://www.tao.lu/Ontologies/generis.rdf#is_statistical';
+    public const PROPERTY_IS_EDITABLE = 'http://www.tao.lu/Ontologies/generis.rdf#is_editable';
```
Comment (gabrielfs7) [inline core/GenerisRdf.php:34]: Why we need this to correct that "some" properties are not synched to datastore?

Comment (shpran) [reply core/GenerisRdf.php:34]: We need this one to mark lists as non-editable for local lists in UI.

File: migrations/Version202411200904082348_generis.php (hunk 1 above)
Comment (shpran) [reply migrations/Version202411200904082348_generis.php:23]: Did we already implement this feature?

**Comment by shpran** [comment]
Not needed anymore

**Pull request closed without merging**